.circleci/config.yml

```diff
@@ -28,9 +28,10 @@ jobs:
     working_directory: ~/marquez_python
     steps:
       - checkout
+      - run: ./docker/release_check.sh
```

Comment:
Do we want this script to live on under `docker/`? Also, do we need a `docker/` if we don't have a `Dockerfile` defined?